### PR TITLE
Create 1

### PR DESCRIPTION
**Pull request by samspeer**
Created the game page and updated formatting

### CODE REVIEW

File: client/src/components/create.tsx [hunk 1]
```diff
@@ -2,34 +2,36 @@
...
-  // padding-top: 160px;
   padding-top: 0px;
   padding-bottom: 0px;
 `;
 
-const gameModes: { name: string, description: string, src: string }[] = [
-    { name: "Solo", "description": "Practice Math On Your Own", "src": "/images/solo.png" },
-    { name: "Head to Head", "description": "Play With A Randomly Matched Foe", "src": "head.png" },
-    { name: "Group Play", "description": "Play With 2+ Friends In A Private Room", "src": "group.png"  }
+const gameModes: { name: string, description: string, icon: any }[] = [
```
Comment (timothycobrien) [inline client/src/components/create.tsx:31]: Make this type line up with the possible types of icon using a union type or interface

File: client/src/index.tsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ import {CenteredDiv, HeaderWrap} from "./components/shared";
 
 const GridBase = styled.div`
   positive: relative;
+  font-family: Revalia;
```
Comment (timothycobrien) [inline client/src/index.tsx:22]: Rather use a different font, and add backup font possibly

**Review by timothycobrien — CHANGES_REQUESTED** [review]

File: client/src/components/create.tsx (hunk 1 above)
Comment (timothycobrien) [reply client/src/components/create.tsx]: `icon` should not have `any` type

Comment (samspeer) [reply client/src/components/create.tsx]: I am not quite sure how to type the fact it is a component?

Comment (timothycobrien) [reply client/src/components/create.tsx]: It's either a PersonIcon or a GroupIcon right, so type it as the union of these two types so it can be either

**Review by timothycobrien — CHANGES_REQUESTED** [review]

**Review by timothycobrien — APPROVED** [review]

**Pull request merged**